### PR TITLE
Update installation.md, consolidate Fedora infos

### PR DESCRIPTION
**Pull request by frederik-b**
Consolidated infos for Fedora and CentOS which appeared twice.

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -281,8 +280,13 @@ sudo apt-get -qq -y install podman
 
 #### Fedora
 
-You can test the very latest Podman in Fedora's `updates-testing`
-repository before it goes out to all Fedora users.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:285]: I would like to keep this line still, but move it to 289.5, just below where you moved the first line here.

File: getting-started/installation.md [hunk 2]
```diff
@@ -281,8 +280,13 @@ sudo apt-get -qq -y install podman
...
+Podman is available in the `updates` repository.
+
+```bash
+sudo yum -y install podman
+```
+
+You can test the very latest Podman in Fedora's `updates-testing`.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:289]:
```suggestion
You can test the very latest Podman in Fedora's `updates-testing`
repository before it goes out to all Fedora users.
```

File: getting-started/installation.md [hunk 3]
```diff
@@ -281,8 +280,13 @@ sudo apt-get -qq -y install podman
 
 #### Fedora
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:281]:
```suggestion
#### [Fedora](https://www.fedoraproject.org)
```

**Comment by TomSweeneyRedHat** [comment]
@lsm5 PTAL

File: getting-started/installation.md [hunk 4]
```diff
@@ -281,8 +280,13 @@ sudo apt-get -qq -y install podman
...
 
-You can test the very latest Podman in Fedora's `updates-testing`
-repository before it goes out to all Fedora users.
+Podman is available in the `updates` repository.
+
+```bash
+sudo yum -y install podman
```
Comment (nasirhm) [inline getting-started/installation.md:286]: `sudo dnf install -y podman` as [dnf](https://en.wikipedia.org/wiki/DNF_(software)) is the default package manager for Fedora.

**Review by nasirhm — CHANGES_REQUESTED** [review]
Nice work on it.

**Comment by lsm5** [comment]
@frederik-b could you rebase please?

**Comment by rhatdan** [comment]
Most of this is already fixed. Closing.

**Pull request closed without merging**